docs/UserGuide.adoc

```diff
@@ -1021,7 +1037,7 @@ e.g. `addpolicy n/Senior Care d/care for seniors c/months/10 pr/$50000 [sa/50 ea
 * *List Bin Items* : `listbin`
```

Comment:
Can you help to change the addpolicy command a few lines above? There's an additional `addpolicy` on line 1017/1033 and the square brackets should be removed for `[sa/50 ea/75]` on the next line.